scripts/mm2f.ps1

```
@@ -0,0 +1,86 @@
+        continue
+    }
+
+    $template = $commands.$pm
+    if (-not $template) {
+        $template = $defaultCommands[$pm]
+    }
+
+    $cmd = $template -replace '\{id\}', $id
+
+    Write-Host "Installing $id ..." -ForegroundColor Cyan
+    Invoke-Expression $cmd
```

PSScriptAnalyzer (1.25.0)

[warning] Invoke-Expression is used. Please remove Invoke-Expression from script and find other options instead.

(PSAvoidUsingInvokeExpression)